### DISCUSSION TITLE
delete existing repo

### DISCUSSION THREAD

**Discussion by mishraTeaches** [community/community#22885] [New to GitHub]
<p>How i can delete my existing repo ?</p>

**Comment by Wabri** [comment]
<p>Open the repo then go to settings and the last option is ‘delete this repository’ and you have to type the name of the repo.</p>

**Comment by waagyi** [comment]
<p>I could not find the delete repo…</p>
<p>I opened the repo. . .</p>
<p>Then go to setting. . .</p>
<p>Please help me.I read the stack overflow and they say the same luke you.</p>

**Comment by mpboom** [comment, marked as answer]
<p><a href="https://help.github.com/articles/deleting-a-repository/" rel="nofollow noopener">https://help.github.com/articles/deleting-a-repository/</a></p>
